content/doc/tutorials/create-a-pipeline-in-blue-ocean.adoc

```diff
@@ -8,430 +8,294 @@ section: doc
-. Click *Create Pipeline*. +
-  Blue Ocean detects that there is no Jenkinsfile at the root level of the
-  repository's `master` branch and proceed to help you create one. (Therefore,
-  you'll need to click another *Create Pipeline* at the end of the page to
-  proceed.) +
-  *Note:* Under the hood, a Pipeline project created through Blue Ocean is
-  actually "multibranch Pipeline". Therefore, Jenkins looks for the presence of
-  at least one Jenkinsfile in any branch of your repository.
+image:tutorials/pipeline-in-blueocean-01-connecting-to-github.png[alt="Connecting to GitHub",width=80%]
++
+[[choose-github-account]]Jenkins now has access to your GitHub account, using your access token.
+
+. In *Which organization does the repository belong to?*, select your GitHub account where you forked the repository <<fork-sample-repository,above>>.
+. In *Choose a repository*, select your forked repository `creating-a-pipeline-in-blue-ocean`.
+. Select *Create Pipeline*.
+* Blue Ocean detects that there is no Jenkinsfile at the root level of the repository's `master` branch and helps you create one.
+You'll need to click another *Create Pipeline* at the end of the page to proceed.
 
+NOTE: Under the hood, a Pipeline project created through Blue Ocean is actually "multibranch Pipeline".
+Therefore, Jenkins looks for the presence of at least one Jenkinsfile in any branch of your repository.
 
 === Create your initial Pipeline
 
-. Following on from creating your Pipeline project
-  (<<create-your-pipeline-project-in-blue-ocean,above>>), in the Pipeline
-  editor, select *docker* from the *Agent* dropdown in the *Pipeline Settings*
-  panel on the right. +
+. Following on from creating your Pipeline project <<create-your-pipeline-project-in-blue-ocean,above>>, in the Pipeline editor, select *docker* from the *Agent* dropdown in the *Pipeline Settings* panel on the right.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-11-initial-pipeline-editor-page.png[alt="Initial
-to GitHub",width=100%]
-. In the *Image* and *Args* fields that appear, specify `node:6-alpine` and
-  `-p 3000:3000` respectively. +
+image:tutorials/pipeline-in-blueocean-11-initial-pipeline-editor-page.png[alt="Initial to GitHub",width=100%]
+. In the *Image* and *Args* fields that appear, specify `node:6-alpine` and `-p 3000:3000` respectively.
++
 [.boxshadow]
-image:tutorials/pipeline-in-blueocean-12-agent-section.png[alt="Configuring the
-agent",width=50%] +
-  *Note:* For an explanation of these values, refer to annotations *1* and *2*
-  of the Declarative Pipeline in the
-  link:../build-a-node-js-and-react-app-with-npm/#create-your-initial-pipeline-as-a-jenkinsfile[``Create
-  your initial Pipeline...'' section of the Build a Node.js and React app]
-  tutorial.
-. Back in the main Pipeline editor, click the *+* icon, which opens the new
-  stage panel on the right. +
+image:tutorials/pipeline-in-blueocean-12-agent-section.png[alt="Configuring the agent",width=50%]
++
```

Comment:
I feel a little silly if these aren't needed. Maybe just test the page for how they render if there is an extra blank line to see if it is needed, and update each accordingly. I won't call out any more of these for this page, so as not to spam you with something that may not be needed. 
```suggestion

+
```